### PR TITLE
ref(ui): Convert .tag-bar to emotion

### PR DESCRIPTION
**Pull request by scttcper**
its these progress bar things

before
image

after with new theme colors. Shown with hover state on 2nd element.
Screen Shot 2021-04-07 at 11 46 13 AM
Screen Shot 2021-04-07 at 11 46 03 AM

### CODE REVIEW

File: src/sentry/static/sentry/app/views/organizationGroupDetails/groupTags.tsx [hunk 1]
```diff
@@ -176,4 +176,33 @@ const TagItem = styled('div')`
...
+  margin-bottom: ${space(0.5)};
+  padding: 0 ${space(1)};
+`;
+
+const TagBarBackground = styled('div')`
+  position: absolute;
+  top: 0;
+  bottom: 0;
+  left: 0;
+  background: #f6f7f8;
```
Comment (scttcper) [inline src/sentry/static/sentry/app/views/organizationGroupDetails/groupTags.tsx:193]: theme color?

Comment (billyvg) [reply src/sentry/static/sentry/app/views/organizationGroupDetails/groupTags.tsx:193]: yes

**Comment by billyvg** [comment]
Is there something in dashboards that is similar?

**Review by robinrendle — COMMENTED** [review]
Few color suggestions here to improve the background:

CleanShot 2021-04-06 at 10 29 27@2x (1)
CleanShot 2021-04-06 at 10 27 05@2x (1)

File: src/sentry/static/sentry/app/views/organizationGroupDetails/groupTags.tsx (hunk 1 above)
Comment (markstory) [reply src/sentry/static/sentry/app/views/organizationGroupDetails/groupTags.tsx:193]: I would normalize on something in theme. There are some chart related colours that map to grey200 in lightmode that could work here.

File: src/sentry/static/sentry/app/views/organizationGroupDetails/groupTags.tsx [hunk 2]
```diff
@@ -104,19 +105,18 @@ class GroupTags extends React.Component<Props, State> {
...
+              <TagBarGlobalSelectionLink
                 to={{
                   pathname: `${baseUrl}events/`,
                   query: {query},
                 }}
               >
-                <span className="tag-bar-background" style={{width: pct + '%'}} />
-                <span className="tag-bar-label">{label}</span>
-                <span className="tag-bar-count">
+                <TagBarBackground style={{width: pct + '%'}} />
```
Comment (priscilawebdev) [inline src/sentry/static/sentry/app/views/organizationGroupDetails/groupTags.tsx:114]: should we move this style to emotion?

**Comment by priscilawebdev** [comment]
it would be nice to have a screenshot of this UI in dark mode too :)

**Comment by scttcper** [comment]
@priscilawebdev updated with new screenshots

**Comment by scttcper** [comment]
updated again with new hover state thanks @robinrendle

**Review by billyvg — APPROVED** [review]

**Pull request merged**